framework/db/mapping_manager.py

```diff
@@ -74,6 +74,13 @@ def GetMappingTypes(self):
         """
         return self.mapping_types
 
+    def GetAllMappings(self):
```

Comment:
Are we changing case system because all other function names are in `camel case` ?